### PR TITLE
[13.0][MIG] connector_elasticsearch

### PR DESCRIPTION
**Pull request by rousseldenis**
Includes :

* [x] Forward port of 12.0 branch

### CODE REVIEW

**Review by lmignon — APPROVED** [review]

**Comment by rousseldenis** [comment]
@sebastienbeau @guewen

File: connector_algolia/components/adapter.py [hunk 1]
```diff
@@ -45,13 +62,19 @@ def index(self, records):
...
+        index = self.get_index()
         index.delete_objects(binding_ids)
 
     def clear(self):
-        index = self._get_index()
+        index = self.get_index()
         index.clear_index()
+        self.settings(force=True)
 
     def iter(self):
```
Comment (simahawk) [inline connector_algolia/components/adapter.py:73]: can't we remove `iter` now?

**Review by simahawk — APPROVED** [review]

File: connector_algolia/components/adapter.py (hunk 1 above)
Comment (simahawk) [reply connector_algolia/components/adapter.py:73]: you can do it later....

**Pull request merged**